salt/utils/thin.py

```diff
@@ -292,6 +292,16 @@ def gen_thin(cachedir, extra_mods='', overwrite=False, so_mods='',
             if tempdir is not None:
                 shutil.rmtree(tempdir)
                 tempdir = None
+
+        # workaround for backports
+        os.symlink('salt/ext','/tmp/workaround_symlink_backports')
```

Comment:
When we create this file it should be in a secure tempdir to prevent someone from injecting code from an unprivieged user account. So make a subdir first with the python tempfile module, and then make the symlink in there, then remove the tempdir when you are done with shutil.rmtree